keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/resource/TestUtils.kt

```diff
@@ -0,0 +1,41 @@
+package com.netflix.spinnaker.keel.titus.resource
+
+import com.netflix.spinnaker.keel.api.titus.CLOUD_PROVIDER
+import com.netflix.spinnaker.keel.api.titus.cluster.TitusServerGroup
+import com.netflix.spinnaker.keel.api.titus.cluster.moniker
+import com.netflix.spinnaker.keel.clouddriver.model.Placement
+import com.netflix.spinnaker.keel.clouddriver.model.SecurityGroupSummary
+import com.netflix.spinnaker.keel.clouddriver.model.ServiceJobProcesses
+import com.netflix.spinnaker.keel.clouddriver.model.TitusActiveServerGroup
+import com.netflix.spinnaker.keel.clouddriver.model.TitusActiveServerGroupImage
+import com.netflix.spinnaker.keel.model.parseMoniker
+import org.apache.commons.lang3.RandomStringUtils
+
+fun TitusServerGroup.toClouddriverResponse(
```

Comment:
Pulling this out to be used by both export and cluster handler tests. No change in functionality.